codegen/java-gen/src/test/scala/com/lightbend/akkasls/codegen/java/SourceGeneratorSuite.scala

```diff
@@ -131,6 +131,7 @@ class SourceGeneratorSuite extends munit.FunSuite {
                 generatedSourceDirectory.resolve("com/example/service/domain/MyValueEntity2Provider.java"),
                 generatedSourceDirectory.resolve("com/example/service/domain/MyValueEntity2Router.java"),
                 generatedTestSourceDirectory.resolve("com/example/service/domain/MyValueEntity2TestKit.java"),
+                testSourceDirectory.resolve("com/example/service/domain/MyValueEntity2Test.java"),
```

Comment:
Previous code omitted that file from the returned list of paths (potentially leading to problems when used like this from sbt).